src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs

```diff
@@ -0,0 +1,274 @@
+use crate::{
+    jormungandr::explorer::{
+        all_stake_pools::AllStakePoolsTipAllStakePools, data::stake_pool::StakePoolStakePool,
+        verifiers::ExplorerVerifier,
+    },
+    utils::StakePool,
+};
+use chain_impl_mockchain::transaction::AccountIdentifier;
+use chain_time::TimeOffsetSeconds;
+use std::num::NonZeroU64;
+
+impl ExplorerVerifier {
+    pub fn assert_stake_pool(
+        stake_pool: &StakePool,
+        explorer_stake_pool: &StakePoolStakePool,
+        retirement_time: core::option::Option<TimeOffsetSeconds>,
+    ) {
+        assert_eq!(explorer_stake_pool.id, stake_pool.id().to_string());
+        assert_eq!(
+            explorer_stake_pool.registration.management_threshold as u8,
+            stake_pool.info().management_threshold()
+        );
+
+        assert_eq!(
+            stake_pool.info().operators.len(),
+            explorer_stake_pool.registration.operators.len()
+        );
+
+        assert_eq!(
+            stake_pool.info().operators.len(),
+            explorer_stake_pool.registration.operators.len()
+        );
+        let mut operators_matching = 0;
+        for operator in &explorer_stake_pool.registration.operators {
```

Comment:
```
assert(explorer_stake_pool.registration.operators.iter().all(|operator| stake_pool
                .info()
                .operators
                .contains(&Self::decode_bech32_pk(operator))));
```

Reply:
Nice one, that's what I was looking for!